spec/features/passenger_filters_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+feature 'Passenger Filters' do 
```

Comment:
In general these tests don't ensure that the correct passenger is shown, just that the correct number is shown. It'd be nice to be sure that exactly the right data is present.